zerver/webhooks/sentry/doc.md

```diff
@@ -1,38 +1,36 @@
+# Zulip Sentry integration
+
 Get Zulip notifications for the issues in your Sentry projects!
 
 This integration supports Sentry's Node, Python, and Go
 [platforms](https://sentry.io/platforms/).  [Contact
 us](/help/contact-support) if a platform you care about is missing.
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
+
+    !!! tip ""
 
-    The default topic, if not set in the URL, will be the title of the
-    issue or event.
+        The default topic, if not set in the URL, will be the title of the
+        issue or event.
 
 1. In Sentry, go to your organization's **Settings**, and then go to
     **Developer Settings**. Click on the **Create New Integration** button,
     and select **Internal Integration**. Set the **Webhook URL** to the URL
-    you constructed in the above step, and then enable **Alert Rule
-    Action**.
+    you generated in the above step, and then enable **Alert Rule
+    Action**. You can fill out the remaining details as you like.
 
-    You can fill out the remaining details as you like. If you want
-    notifications for issues and not just events, you can scroll down to
-    **Webhooks** on the same page, and check the box that says **issue**.
-    Make sure that you set up the permissions so that the integration will
-    be visible to the right people.
+    !!! tip ""
 
-    !!! warn ""
-
-        **Note:** Zulip also supports configuring this as a webhook in
-        Sentry &mdash; which, while easier to configure (navigate to
-        **Settings &gt; Integrations &gt; WebHooks**), may not include the
-        full breadth of event types. For instance, some events, like issue
-        assignments or issues being resolved, will not trigger notifications
-        with this configuration.
+        If you want notifications for issues and not just events, you can
+        scroll down to **Webhooks** on the same page, and check the box that
+        says **issue**. Make sure that you set up the permissions so that
+        the integration will be visible to the right people.
 
 1. Once you've saved the internal integration, go to **Alerts** and click
```

Comment:
Move the Save instruction to the previous point.
Start this one from "Go to **Alerts**"